File: XenparkBlankTemplate/Entity/Master.cs
public partial class Master
{
    public int Id { get; set; }
    public string Code { get; set; }
    public string Description { get; set; }
    public string Column1 { get; set; }
    public string Column2 { get; set; }
    public string Column3 { get; set; }
    public int ParentId { get; set; }
    public string ParentCode { get; set; }
    public string ParentDescription { get; set; }
    public bool Approved { get; set; }
}

File: XenparkBlankTemplate/ClientApp/src/models/master.ts
export interface IMaster {
    Id: number;
    Code: string;
    Description: string;
    ParentId: number;
    ParentCode: string;
    ParentDescription: string;
    Approved: boolean;
    Column1:string; // UOM for Product, IP Address for Room
    Column2:string;
    Column3:string;
    IsUsed: boolean;
}

export interface IRoomMaster {
    RoomId: number;
    RoomCode: string;
    RoomDescription: string;
    AreaId: number;
    AreaCode: string;
    AreaDescription: string;
    BlockId: number;
    BlockCode: string;
    BlockDescription: string;
    PlantId: number;
    PlantName: string;
    PlantLocation: string;
}

